packages/web-components/src/components/ic-breadcrumb-group/ic-breadcrumb-group.tsx

```diff
@@ -31,6 +31,8 @@ export class BreadcrumbGroup {
   private collapsedBreadcrumbs: HTMLIcBreadcrumbElement[];
   private ADD_CLASS_DELAY = 50;
   private collapsedBreadcrumbWrapper: HTMLIcBreadcrumbElement;
+  private showBackIconString: string = "show-back-icon";
```

Comment:
I think we should consider using UPPERCASE for `const` that won't change. Similar to line 32. https://javascript.info/variables#uppercase-constants

Probably won't need to add 'String' at end as well.